humanguide/design-concepts/app-icon-design.rst

```diff
@@ -0,0 +1,109 @@
+App icon design
+===============
+
+The whole system and apps follow the `Suru design principles <http://web.archive.org/web/20191223082720/https://docs.ubuntu.com/phone/en/apps/design/get-started/design-philosophy>`__ started by Canonical and `Samuel Hewitt <https://icons.samuelhewitt.com/>`__.
+
+Design Philosophy and Suru Meaning
+----------------------------------
+
+-  Suru alludes to Japanese culture.
+-  The design of Suru is inspired by origami.
+-  Paper can be used in two and three dimensional forms, because it is
+   transferable and diverse.
```

Comment:
It may be better to flip this sentence. Right now it says that paper is used in 2D or 3D because it is transferable. I think you mean to say that paper is transferable and diverse because it can be folded in many shapes.